### ISSUE TITLE
[Security] Image is using vulnerable version of alpine 3.12.0 

### ISSUE THREAD

**Issue by arieltorti** [fholzer/docker-nginx-brotli#44]
Currently the image is built on top of Alpine 3.12.0 which contains many reported vulnerabilities, including critical ones.
Image should be rebuilt with the latest Alpine 3.12.9.

Running [trivy](https://github.com/aquasecurity/trivy) on fholzer/nginx-brotli:v1.19.1 outputs:
```
trivy fholzer/nginx-brotli:v1.19.1                                
2021-12-20T11:41:52.638-0300	INFO	Detected OS: alpine
2021-12-20T11:41:52.638-0300	INFO	Detecting Alpine vulnerabilities...
2021-12-20T11:41:52.640-0300	INFO	Number of language-specific files: 0

fholzer/nginx-brotli:v1.19.1 (alpine 3.12.0)
============================================
Total: 52 (UNKNOWN: 0, LOW: 2, MEDIUM: 17, HIGH: 30, CRITICAL: 3)
```

**Comment by enricobenedos** [comment]
Latest version is referencing `alpine:3.15`

**Comment by fholzer** [comment]
closed in #45
